WikiTemplate/Crypto/wikitemplate-cryptoiOS.md

```diff
@@ -0,0 +1,60 @@
+
+## Onboarding
+
+- [ ] Verify able to create a wallet by following onboarding
+- [ ] Verify not able to screenshots during onboarding screens
+- [ ] Verify able to backup wallet seed phrase during onboarding
+- [ ] Verify able to skip backup wallet seed screen to complete onboarding
+- [ ] Verify onboarding is shown when trying to interact with Dapp without creating wallet
+
+## Panel
+
+- [ ] Verify panel is only shown in URL when a Dapp is trying to interact
+- [ ] Verify able to change default network from panel
+- [ ] Verify able to change accounts from panel
+- [ ] Verify notification dot is shown when panel when an action is pending for Wallet
+
+## Ethereum
+
+- [ ] Verify Ethereum account is created as soon as onboarding is completed
+- [ ] Verify able to create Ethereum account can be created from accounts tab
+- [ ] Verify able to import Ethereum account via Private Key/JSON file
+- [ ] Verify able to submit an ETH transaction
+
+## Solana
+
+- [ ] Verify able to create a Solana account when Solana Mainnet Beta is selected from network picker
+- [ ] Verify able to create a Solana account from accounts tab
+- [ ] Verify able to send tokens from one Solana account to another
+- [ ] Verify able to submit a SOL transaction
+
+## Network Settings
+
+- [ ] Verify able to add a custom network via Settings
+- [ ] Verify able to change active network to a custom network via portfolio
+
+## Custom Tokens
+
+- [ ] Verify able to manually add a custom token
+- [ ] Verify able to manually add ERC721 tokens
```

Comment:
iOS does not yet support adding custom ERC721 tokens.

Reply:
Added comment that its not implemented. Will just have to remove comment when its done